spec/fixtures/authorities/linked_data/lod_lang_defaults.json

```diff
@@ -18,7 +18,7 @@
       "term_id": "term_id"
     },
     "term_id": "ID",
-    "language": [ "en" ],
+    "language": [ "fr" ],
```

Comment:
Change this so the authority config language would be different than the Qa default language.